index.html

```diff
@@ -14084,7 +14085,12 @@ <h3>Acknowledgments</h3>
 			<!-- list of contributors will appear here,
 					 along with link to their GitHub profiles -->
 		</ul>
-		<div data-include="common/acknowledgements/aria-wg-active.html" data-include-replace="true"></div>
+		<section class="section" id="ack_group">
+			<h4>Participants active in the ARIA WG at the time of publication</h4>
```

Comment:
As per discussion on the editors' call today, let's remove  "active".

Reply:
Should be done now.